neptune/new/internal/utils/__init__.py

```diff
@@ -170,3 +174,13 @@ def is_ipython() -> bool:
         return ipython is not None
     except ImportError:
         return False
+
+
+def expect_not_an_experiment(run: 'Run'):
+    from neptune.new.exceptions import NeptuneLegacyIncompatibilityException
```

Comment:
Why not at the beginning of the file?

Reply:
not proud of it, but this deals with circular deps (`exceptions.py` uses `utils/__init__.py`)

Reply:
Ok